.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    permissions:
+      contents: write
+      packages: write
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: 'go.mod'
+
+    - name: Run GoReleaser
```

Comment:
I'll have to modify our recipe in https://blog.aspect.dev/releasing-bazel-rulesets-rust a bit to account for this tool wanting to build the artifacts and ALSO create the release and publish the artifacts to it. Probably not too hard...